acquia_lift.routing.yml

```diff
@@ -4,4 +4,47 @@ acquia_lift.admin_settings_form:
     _form: '\Drupal\acquia_lift\Form\AdminSettingsForm'
     _title: 'Acquia Lift settings'
   requirements:
-    _permission: 'administer acquia lift'
+    _permission: 'administer acquia_lift'
+
+acquia_lift.slot.overview:
+  path: '/admin/config/content/acquia-lift/slots'
```

Comment:
I would recommend to do "acquia_lift.slot.overview" and '/admin/config/content/acquia-lift/slot/overview'. The minimum discrepancy between the route id and actual path has helped me with greatly reducing parsings in my past.